### DISCUSSION TITLE
Expand target row on Drag Start(Hovering)

### DISCUSSION THREAD

**Discussion by Ashokkumar1995** [IgniteUI/igniteui-angular#9107] [Q&A]
When i drag a row and hover it on the another row(parent) which is collapsed, i dont see any event to find what is the row which i hover right now.

i can only find the target row only after dropping it.

is there a way to find it while dragging ???

**Comment by dafo** [comment]
@Ashokkumar1995 
Can you please check [this example](https://stackblitz.com/edit/igx-grid-row-drag-highlight?file=src%2Fapp%2Fgrid%2Fgrid-row-reorder-sample%2Fgrid-row-reorder.ts).
It demonstrates how you can add a visual clue where a row will be dropped by checking the currently hovered element, taking advantage of the "[over](https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/igxdropdirective.html#over)" emitter.

**Reply by Ashokkumar1995** [reply]
yeah i tried, but still i cant achieve it.

**Comment by zdrawku** [comment]
@Ashokkumar1995 with the latest releases we've [introduced a couple of Row Public API enhancements](https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/update-guide#igxgridcomponent-igxtreegridcomponent-igxhierarchicalgridcomponent), now we return object implementing the public [RowType](https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/interfaces/rowtype.html) interface. 

Could you please try it out, if something is missing from the usacase that you have, we can consider adding it.
